vars/Debian.yml

```diff
@@ -10,15 +10,16 @@ hwraid_apt_le_vert_keys:
   - http://hwraid.le-vert.net/debian/hwraid.le-vert.net.gpg.key
 
 # HP apt repository
-hwraid_apt_hp_repository: 'deb http://downloads.linux.hpe.com/SDR/repo/mcp/{{ ansible_distribution | lower }} {{ ansible_distribution_release | lower }}/current non-free'
+hwraid_apt_hp_repository: 'deb http://downloads.linux.hpe.com/SDR/repo/mcp/{{ ansible_distribution | lower }} {{ ansible_distribution_release | lower }}/{{ hwraid_hp_repository_version }}'
 
 # hwraid.le-vert.net apt repository
 hwraid_apt_le_vert_repository: 'deb http://hwraid.le-vert.net/{{ ansible_distribution | lower }} {{ ansible_distribution_release | lower }} main'
 
 # HP raid controller tools
 hwraid_hp_raid_packages:
-  - hpacucli
-  - hpssacli
+  - ssa
+  - ssacli
+  - ssaducli
```

Comment:
You did define them in RedHat.yml so I chose to allign with what was defined there ;)

* ssacli being a (almost) direct replacement for hpacucli should be present on any system with a HPE RAID conterollers.
* ssaducli looks pretty sane to me as well since it is the diagnostics utility for them.
* ssa is the one I'm a bit reluctant to say yes. I've both never used it, nor have I found yet the thing making it essential to us.

So far, if a HP(E) volume is detected we can mostly cound on HPE to support them with these tools if they support it by providing required packages for the required OS/HW combination. (unlike megacli/storcli/sas[2|3]ircu from LSI/Broadcom)

I don't think we can count on HPE to keep package names between Debian-based and EL distribution identical. We can likely assume the package names are the same for Debian distributions or EL distributions, but not both using the same package naming.